scripts/index.js

```diff
@@ -21,32 +21,36 @@ function handleAdd(e){
                 break;
             }
         }
-            if(m==1){
-               
-                document.getElementById("dupli").innerHTML ="<div class='alert alert-danger'>Title already exists. </div>";
-                document.getElementById("success").innerHTML ="";
-            }
-            else{
-                document.getElementById("success").innerHTML ="<div class='alert alert-success'> Title added successfully </div>";
-                document.getElementById("dupli").innerHTML ="";
-                localStorage.setItem(i, addTxt.value);
-                if(localStorage.length ==1) titles.removeChild(titleMessage);
-                insertCard(addTxt.value);
-                addTxt.value="";
-                i++;
-            }
-    }
-    else
-    {
-        console.log()
-        document.getElementById("add-txt").blur();
-        addTxt.classList.add('active');
-        setTimeout(RemoveClass, 1000);
-        addTxt.placeholder="Cannot Be Blank!!!!";
-        function RemoveClass() {
-        document.getElementById("add-txt").focus();
-        addTxt.classList.remove('active');
-        addTxt.placeholder="Enter title";
+            
+    
+        if(m==1){
+           
+            document.getElementById("dupli").innerHTML ="<div class='alert alert-danger'>Title already exists. </div>";
+            document.getElementById("success").innerHTML ="";
+            setTimeout(function() {document.getElementById('dupli').innerHTML='';},3000);
+        }
+        else{
+            document.getElementById("success").innerHTML ="<div class='alert alert-success'> Title added successfully </div>";
+            document.getElementById("dupli").innerHTML ="";
+            setTimeout(function() {document.getElementById('success').innerHTML='';},3000);
+            localStorage.setItem(i, addTxt.value);
+            if(localStorage.length ==1) titles.removeChild(titleMessage);
+            insertCard(addTxt.value);
+            addTxt.value="";
+            i++;
+        }
+//     else
+//     {
+//         console.log()
+//         document.getElementById("add-txt").blur();
+//         addTxt.classList.add('active');
+//         setTimeout(RemoveClass, 1000);
+//         addTxt.placeholder="Cannot Be Blank!!!!";
+//         function RemoveClass() {
+//         document.getElementById("add-txt").focus();
+//         addTxt.classList.remove('active');
+//         addTxt.placeholder="Enter title";
+
```

Comment:
@ankichd11 why have you commented this, this was responsible for generating error messages when a user enters a blank title?

Reply:
The else there was inappropriate as it was not letting the items to add.  On the other hand whoever so didi this commented out the block which was adding items.

Reply:
The issue is resolved